Documentation/development-environment.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Multi-Node Test Environment
+weight: 91
+indent: true
+---
+
+# Rook development workflow
+
+## Setup expectation
+
+There are a bunch of pre-requisites to be able to deploy the following environment. Such as:
```

Comment:
It would be nice if we had pointers to start different types of multi-node clusters so in the future this topic might be split into multiple. This might be the kubespray example and another could be to start with kubeadm, etc. But no need to change this yet.